tests/helper_functions.py

```diff
@@ -616,3 +616,10 @@ def clear_hub_cache():
 
 def get_free_disk_space_in_GiB():
     return shutil.disk_usage("/").free / (1024**3)
+
+
+def create_random_array():
+    a = []
+    for i in range(3):
+        a.append(i)
+    return a
```

Comment:
$ai Simplify this function.

Reply:
$ai You're a code reviewer. If you suggest code, please make sure to use the following format.

`````
```suggestion
<code>
```
`````

Let's simplify this function using list comprehension.